CHANGELOG.md

```diff
@@ -43,6 +44,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Change http code for DecommissioningFailedException from 500 to 400 ([#5283](https://github.com/opensearch-project/OpenSearch/pull/5283))
 - Pre conditions check before updating weighted routing metadata ([#4955](https://github.com/opensearch-project/OpenSearch/pull/4955))
 - Support remote translog transfer for request level durability([#4480](https://github.com/opensearch-project/OpenSearch/pull/4480))
```

Comment:
Lets fix the change log to include what has been added

Reply:
Added https://github.com/opensearch-project/OpenSearch/pull/5392 for  Remote Translog Functionality

Reply:
@gbbafna  changelog from https://github.com/opensearch-project/OpenSearch/pull/5699/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed is also not there. can you add that?

Reply:
`Add support for refresh level durability` is added under `Added`, we can remove it from here. Also, we can remove segment store integration in failover flow entry.